Segments/Bloodlands.mapproj

```diff
@@ -223173,7 +223216,7 @@
         
         MaxHealth = 8000, Health = 8000,
         BaseDodge = 34,
-        MaxMana = 50, Mana = 50,
```

Comment:
Didn't know if you meant to change this? No worries if so.

Reply:
need to change both maxmana, and mana for this to accomplish anything.   start with 45 then regen back to 50 instead of starting at 50